docs/deploying.md

```diff
@@ -39,7 +39,7 @@ To stop your registry, you would:
 
 ## Storage
 
-By default, your registry data is persisted as a [docker volume](/engine/userguide/containers/dockervolumes.md) on the host filesystem. Properly understanding volumes is essential if you want to stick with a local filesystem storage.
+By default, your registry data is persisted as a [docker volume](https://docs.docker.com/engine/userguide/containers/dockervolumes/) on the host filesystem. Properly understanding volumes is essential if you want to stick with a local filesystem storage.
```

Comment:
The Markdown files on GitHub are used to generate the online documentation. This is a bit troublesome for cross-repository links, therefore these links are broken on GitHub, but will work on the https://docs.docker.com website